cmd/serve.go

```diff
@@ -91,6 +91,8 @@ func init() {
 		"",
 		"config file (default is $HOME/.kubernetes-ldap.yaml)")
 
+	RootCmd.PersistentFlags().StringVar(&keypairDir, "keypair-dir", "keypair", "directory that contains keypair for signing/verifying tokens. Defaults to 'keypair'")
```

Comment:
Isn't the default the third argument to StringVar? Why mention it again in the description?